release.md

```diff
@@ -55,78 +50,101 @@
+skopeo copy --all docker://quay.io/scylladb/scylla-operator:latest-YYYY-MM-DD-HHMMSS docker://quay.io/scylladb/scylla-operator:X.Y.0-alpha.I
+```
+```
+skopeo copy --all docker://quay.io/scylladb/scylla-operator:X.Y.0-alpha.I docker://docker.io/scylladb/scylla-operator:X.Y.0-alpha.I
+```
+
+#### From release branch
+```
+skopeo copy --all docker://quay.io/scylladb/scylla-operator:X.Y-YYYY-MM-DD-HHMMSS docker://quay.io/scylladb/scylla-operator:X.Y.Z-beta.I
+```
+```
+skopeo copy --all docker://quay.io/scylladb/scylla-operator:X.Y.Z-beta.I docker://docker.io/scylladb/scylla-operator:X.Y.0-beta.I
+```
+
+#### From a pre-release
+```
+skopeo copy --all docker://quay.io/scylladb/scylla-operator:X.Y.0-rc.I docker://quay.io/scylladb/scylla-operator:X.Y.0
+```
+```
+skopeo copy --all docker://quay.io/scylladb/scylla-operator:X.Y.0 docker://docker.io/scylladb/scylla-operator:X.Y.0
+```
+
+### Creating matching `git` tag
+When the new image is tagged, you should create a matching `git` tag that this image was built from. (This may get automated in the future.)
+```
+revision="$( skopeo inspect '--format={{ index .Labels "org.opencontainers.image.revision" }}' docker://docker.io/scylladb/scylla-operator:X.Y.Z-alpha.I )"
+```
+```
+git fetch git@github.com:scylladb/scylla-operator.git
+git tag -s -a vX.Y.Z-alpha.I "${revision}" -m "vX.Y.Z-alpha.I"
+git push git@github.com:scylladb/scylla-operator.git vX.Y.Z-alpha.I
+```
+CI will automatically create a new release in GitHub and publish the [release notes](#release-notes) there, and it will automatically publish Helm charts.
+
+## Release announcements
+A new release should be published in several channels. This is usually done only for RC and GA releases.
+For GA release we first write a google doc and get it reviewed and approved by at least the PM and the TL.
+
+### GitHub release
+
+CI automatically publishes full release notes, listing every PR merged for this release.
+
+### ScyllaDB Forum
+There is a dedicated section on release notes on ScyllaDB Forum that is being linked from the [ScyllaDB docs](https://www.scylladb.com/product/release-notes).
+It filters posts to `Release notes` category that have `operator-release` tags.
+To publish a release, create a new topic (post) in [Release notes](https://forum.scylladb.com/tags/c/scylladb-release-notes/18/operator-release) category.
 
-1. Submit a PR using `master` as target branch.
+| Title                             | Tags                                     | Content                   |
+| :-------------------------------: | :--------------------------------------: | :-----------------------: |
+| `[RELEASE] Scylla Operator X.Y.Z` | `operator-release`, `operator`, `release`| copy&paste the google doc |
 
-1. (optional) Update the release schedule in `docs/source/release.md`.
 
-1. Submit a PR using `master` as target branch.
+### ScyllaDB User Slack
+Send a post to the `#scylla-operator` channel in ScyllaDB-Users Slack based on the following template.
```

Comment:
IMO we should include notable changes paragraph as well, to promote what is important.

Reply:
seems already included in the forum link - I've extended the description